python/ray/serve/tests/test_deploy_2.py

```diff
@@ -427,5 +427,25 @@ def check_health(self):
         time.sleep(0.1)
 
 
+def test_deploy_bad_working_dir_deployment(serve_instance):
```

Comment:
Do we already have a test for a bad app-level `runtime_env`?

Reply:
We have one in `test_cli`!